stubs/backend/github/github-stubs.yaml

```diff
@@ -53,3 +53,15 @@
     headers:
       content-type: application/json
     file: ./backend/github/jsons/github.user.repos.pull.request.commits.json
+
+# GitHub Commit Data
+# Example: repos/xxx/xxx/commits/commit-sha
+- request:
+    method: GET
+    url: /repos/([\w.-]+\/[\w.-]+)/commits/(\b[0-9a-z]+\b)
+
+  response:
+    status: 200
+    headers:
+      content-type: application/json
+    file: ./backend/github/jsons/github.user.repos.commit.json
```

Comment:
Do we need to rename the json file with `commit-sha`? 
And why we add `user` in the json file name? There is no keyword `user` in the request url.

Reply:
OK, will rename it.
Adding a `user` is a reference to other similar URLs, and we will discuss it with you offline